cache-url/package.json

```diff
@@ -23,13 +26,19 @@
     "url": "https://github.com/ampproject/amp-toolbox/issues"
   },
   "homepage": "https://github.com/ampproject/amp-toolbox#readme",
-  "devDependencies": {
-    "eslint": "5.6.1",
-    "eslint-config-google": "0.10.0",
-    "jasmine": "3.2.0"
-  },
   "dependencies": {
     "mime-types": "^2.1.19",
     "punycode": "^2.1.1"
+  },
+  "devDependencies": {
+    "@ampproject/rollup-plugin-closure-compiler": "^0.7.3",
+    "eslint": "5.6.1",
+    "eslint-config-google": "0.10.0",
+    "jasmine": "3.2.0",
+    "rollup": "^0.66.2",
```

Comment:
lets stay consistent and not use `^` ranges

Reply:
Done, thanks for the review! 😄